Rework contribution section (#350)
the-teachingRSE-project/competencies: competencies.md

We thank all the participants of this community workshop!
Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele,
and Samantha Wittke organised the deRSE23 workshop.
Toby Hodges conceptualised and organised the un-deRSE23 workshop.
We also thank all the participants of this follow-up community workshop!
Jeremy Cohen, Gerasimos Chourdakis, Magnus Hagdorn, Jean-Noël Grad, Jan Philipp Thiele, and Matthias Braun organised the deRSE24 workshop in Würzburg.
We also thank all the participants of this community workshop!
Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele,
Jan Linxweiler, Jean-Noël Grad, and Samantha Wittke contributed the initial draft.
Florian Goth supervised the project and did the project administration.
Jean-Noël Grad designed and implemented the software tooling.
We value the comments by Bernhard Rumpe and Wilhelm Hasselbring on previous drafts
of this work.
Everybody contributed to the final review and editing.

Maybe it's obvious, but as you can guess from having various workshops
spread out over a two year period, and a total list of 23 authors

> the CRediT system is far to generic to adequately describe the contributions of everybody.

Reviewer: ```suggestion
the CRediT system is far too generic to adequately describe the contributions of everybody.
```

Renato Alves quickly jumped in to host the first deRSE23 workshop to take over from a sick organiser.
Matthias Braun contributed early versions of the specialisations and also contributed to the survey.
Leyla Jael Castro contributed to the initial draft of the example career path, and helped us in discussions on metadata.
Gerasimos Chourdakis' contributions to the paper are numerous but he especially helped us in writing first drafts for clarifying

> the relationship of the RSE competencies to the SE competencies.

Author: Great point, and gives us the opportunity to reference it again!

Simon Christ helped us with typesetting, contributed the competencies' symbols, and the spell-checker script.

> Jeremy Cohen drafted the initial introduction and contributed the tables for RSEs in centralised RSE departments(Jean-Noël Grad made them render properly).

Reviewer 2: @CaptainSifff @jngrad not sure what you refer to here, but I invested some effort here... :sweat_smile:  https://github.com/the-teachingRSE-project/competencies/pull/184

> Jeremy Cohen drafted the initial introduction and contributed the tables for RSEs in centralised RSE departments(Jean-Noël Grad made them render properly).

Reviewer 2: We addressed this by extending the overall technical contribution of @jngrad and removing the parenthesis regarding the formatting of the tables.

> Jeremy Cohen drafted the initial introduction and contributed the tables for RSEs in centralised RSE departments(Jean-Noël Grad made them render properly).

Reviewer: He was referring to #19. This was a collaborative effort, where my contributions were limited to advocating for using LaTeX rather than HTML and figuring out which LaTeX typos were crashing the LaTeX processor.

Stephan Druskat contributed parts on proper software citation and publication and sharpened various RSE specialisations.
Fredo Erxleben contributed to early discussions of the paper and contributed the efforts done by Helmholtz.
Jean-Noël Grad contributed initial drafts for the ELIXIR framework and the section on the work of the HPC certification forum.
And, most likely, without him, we wouldn't have a bibliography at all.
Magnus Hagdorn drafted and supervised the ethics and values section for an RSE and made sure that these values are reflected in the competencies of RSEs.
Toby Hodges contributed parts on the Carpentries, and helped steer the curriculum discussion.
Guido Juckeland contributed experiences from his first RSE course for students.
Dominic Kempf drafted the first version of the example curriculum.

> Anna-Lena Lamprecht helped us to use proper wording and make us aware, where we are misusing established SE terms.

Reviewer: ```suggestion
Anna-Lena Lamprecht helped us to use proper wording and made us aware, where we were misusing established SE terminology.
```

Jan Linxweiler drafted various RSE specialisations and made sure that clean coding techniques have their importance.

> Frank Löffler rewrote numerous parts to be actually legible, and helped us in steering onto the final meters for a de-RSE position paper.

Reviewer: is "final meters" used outside of sports? I could certainly be wrong, but in the context of project management, "the last mile" and "the final stretch" feel more natural to me.

Author: :-) Yes, you're right. Additionally it's a literal german translation, since we don't have last miles . Will change!

Reviewer 2: "last steps" would surprise nobody in this context, I think (unless you want to emphasize that these were too many or large steps).

that provide this education will be the topic of a follow-up paper.

# Contribution details {-}
Heidi Seibold came up with the original idea for the deRSE23 workshop in Paderborn.
We thank all the participants of this community workshop!

> Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele,

Reviewer 3: Do we maybe want to change this sentence and the previous, it feels weirdly ordered at the moment with idea, thank participants, organisers.

Author: technically you can say parts of it are redundant. But I didn't want to waste any opportunity to thank our participants....

Author: 2195922 now has that changed. How about it?

Reviewer 3: No, having thanks for each workshop is certainly good!

and Samantha Wittke organised the deRSE23 workshop.
Toby Hodges conceptualised and organised the un-deRSE23 workshop.
We also thank all the participants of this follow-up community workshop!
Jeremy Cohen, Gerasimos Chourdakis, Magnus Hagdorn, Jean-Noël Grad, Jan Philipp Thiele, and Matthias Braun organised the deRSE24 workshop in Würzburg.
We also thank all the participants of this community workshop!
Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele,
Jan Linxweiler, Jean-Noël Grad, and Samantha Wittke contributed the initial draft.
Florian Goth supervised the project and did the project administration.
Jean-Noël Grad designed and implemented the software tooling.
We value the comments by Bernhard Rumpe and Wilhelm Hasselbring on previous drafts
of this work.
Everybody contributed to the final review and editing.

> Maybe it's obvious, but as you can guess from having various workshops

Reviewer 4: ```suggestion
Maybe it is obvious, but as you can guess from having various workshops
```

Matthias Braun contributed early versions of the specialisations and also contributed to the survey.
Leyla Jael Castro contributed to the initial draft of the example career path, and helped us in discussions on metadata.
Gerasimos Chourdakis' contributions to the paper are numerous but he especially helped us in writing first drafts for clarifying
the relationship of the RSE competencies to the SE competencies. He also designed the "Learning and teaching RSE" website [@learnandteach].
Simon Christ helped us with typesetting, contributed the competencies' symbols, and the spell-checker script.
Jeremy Cohen drafted the initial introduction and contributed the tables for RSEs in centralised RSE departments (Jean-Noël Grad made them render properly).
Stephan Druskat contributed parts on proper software citation and publication and sharpened various RSE specialisations.
Fredo Erxleben contributed to early discussions of the paper and contributed the efforts done by Helmholtz.
Jean-Noël Grad contributed initial drafts for the ELIXIR framework and the section on the work of the HPC certification forum.

> And, most likely, without him, we wouldn't have a bibliography at all.

Reviewer 4: ```suggestion
And, most likely, without him, we would not have a bibliography at all.
```

We also thank all the participants of this follow-up community workshop!
Jeremy Cohen, Gerasimos Chourdakis, Magnus Hagdorn, Jean-Noël Grad, Jan Philipp Thiele, and Matthias Braun organised the deRSE24 workshop in Würzburg.
We are also grateful to the participants of this third community workshop!
Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele,
Jan Linxweiler, Jean-Noël Grad, and Samantha Wittke contributed the initial draft.
Florian Goth supervised the project and did the project administration.
Jean-Noël Grad designed and implemented the software tooling.
Everybody contributed to the final review and editing.

The CRediT system [@Brand2015] is far too generic to adequately describe the contributions of everybody in various workshops, spread over a two year period.
While everybody contributed to the entirety of the script, some parts merit special mention.
Renato Alves quickly jumped in to host the first deRSE23 workshop to take over from a sick organiser.
Matthias Braun contributed early versions of the specialisations and also contributed to the survey.

> Leyla Jael Castro contributed to the initial draft of the example career path, and helped us in discussions on metadata.

Reviewer 3: ```suggestion
Leyla Jael Castro contributed to the initial draft of the example career path, and provided helpful insights in discussions on metadata.
```

Reviewer 4: I think this should be insights _to_

Simon Christ helped with typesetting and contributed the competencies' symbols and the spell-checker script.
Jeremy Cohen drafted the initial introduction and contributed the tables for RSEs in centralised RSE departments (Jean-Noël Grad made them render properly).
Stephan Druskat contributed parts on proper software citation and publication and sharpened various RSE specialisations.
Fredo Erxleben contributed to early discussions of the paper and added the contributions by the Helmholtz Association.
Jean-Noël Grad contributed initial drafts for the ELIXIR framework and the section on the work of the HPC certification forum.
And, most likely, without him, we would not have a bibliography at all.
Magnus Hagdorn drafted and supervised the ethics and values section for an RSE and made sure that these values are reflected in the competencies of RSEs.
Toby Hodges contributed parts on the Carpentries, and helped steer the curriculum discussion.
Guido Juckeland contributed experiences from his first RSE course for students.
Dominic Kempf drafted the first version of the example curriculum.

> Anna-Lena Lamprecht helped with proper wording and especially made us aware, where we were misusing established SE terminology.

Reviewer 3: ```suggestion
Anna-Lena Lamprecht helped with proper wording, especially with awareness about established SE terminology, that was misued earlier.
```

> Anna-Lena Lamprecht helped with proper wording, especially with awareness about established SE terminology, that was misued earlier.

Reviewer 4: ```suggestion
Anna-Lena Lamprecht helped with proper wording, especially with awareness about established SE terminology, that was missed earlier.
```
fix typo

Jan Linxweiler drafted various RSE specialisations and made sure that clean coding techniques got their due recognition.

> Frank Löffler rewrote numerous parts to be actually legible, and helped us in preparing for the last mile of a de-RSE position paper.

Reviewer 3: ```suggestion
Frank Löffler rewrote numerous parts to be actually legible, and helped with preparation for the final steps of a de-RSE position paper.
```

that provide this education will be the topic of a follow-up paper.

# Contribution details {-}
Heidi Seibold came up with the original idea for the deRSE23 workshop in Paderborn.
Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele, and Samantha Wittke organised the deRSE23 workshop.
We thank all the participants of this community workshop!
Toby Hodges conceptualised and organised the un-deRSE23 workshop together with Jan Philipp Thiele and Florian Goth.
We also thank all the participants of this follow-up community workshop!
Jeremy Cohen, Gerasimos Chourdakis, Magnus Hagdorn, Jean-Noël Grad, Jan Philipp Thiele, and Matthias Braun organised the deRSE24 workshop in Würzburg.
We are also grateful to the participants of this third community workshop!
Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele,
Jan Linxweiler, Jean-Noël Grad, and Samantha Wittke contributed the initial draft.
Florian Goth supervised the project and did the project administration.
Jean-Noël Grad designed and implemented the software tooling.

> We value the comments by Bernhard Rumpe and Wilhelm Hasselbring on previous drafts
> of this work.

Reviewer 3: ```suggestion

```
Move to `acknowledgements.md`

Everybody contributed to the final review and editing.

The CRediT system [@Brand2015] is far too generic to adequately describe the contributions of everybody in various workshops, spread over a two year period.

> While everybody contributed to the entirety of the script, some parts merit special mention.

Reviewer 2: ```suggestion
While everybody contributed to the discussion formulating and refining the ideas, and to collaboratively writing and/or reviewing and editing the entirety the script, some parts merit special mention.
```

Renato Alves quickly jumped in to host the first deRSE23 workshop to take over from a sick organiser.
Matthias Braun contributed early versions of the specialisations and also contributed to the survey.
Leyla Jael Castro contributed to the initial draft of the example career path, and helped us in discussions on metadata.
Gerasimos Chourdakis' contributions to the paper are numerous but he especially helped us in writing first drafts for clarifying
the relationship of the RSE competencies to the SE competencies. He also designed the "Learning and teaching RSE" website [@learnandteach].

> Simon Christ helped us with typesetting, contributed the competencies' symbols, and the spell-checker script.

Reviewer 5: ```suggestion
Simon Christ helped with typesetting, contributed the competencies' symbols, and the spell-checker script.
```

Jeremy Cohen drafted the initial introduction and contributed the tables for RSEs in centralised RSE departments (Jean-Noël Grad made them render properly).
Stephan Druskat contributed parts on proper software citation and publication and sharpened various RSE specialisations.
Fredo Erxleben contributed to early discussions of the paper and added the contributions by the Helmholtz Association.
Jean-Noël Grad contributed initial drafts for the ELIXIR framework and the section on the work of the HPC certification forum.
And, most likely, without him, we would not have a bibliography at all.
Magnus Hagdorn drafted and supervised the ethics and values section for an RSE and made sure that these values are reflected in the competencies of RSEs.
Toby Hodges contributed parts on the Carpentries, and helped steer the curriculum discussion.
Guido Juckeland contributed experiences from his first RSE course for students.
Dominic Kempf drafted the first version of the example curriculum.

> Anna-Lena Lamprecht helped us to use proper wording and made us aware, where we were misusing established SE terminology.

Reviewer 5: ```suggestion
Anna-Lena Lamprecht helped with proper wording and especially made us aware, where we were misusing established SE terminology.
```

Jan Linxweiler drafted various RSE specialisations and made sure that clean coding techniques got their due recognition.
Frank Löffler rewrote numerous parts to be actually legible, and helped us in preparing for the last mile of a de-RSE position paper.
Michele Martone wrote the first draft of the environmental sustainability section.
Moritz Schwarzmeier drafted the categorisation of the specialisations.
Heidi Seibold contributed the idea and started everything.
Jan Philipp Thiele drafted initial parts of the technical pillar of the RSE competencies, and represented the project on numerous discussions.
Harald von Waldow contributed to initial drafts of the Masters program and contributed his knowledge to the explainability of computer simulations.
Samantha Wittke contributed the parts on CodeRefinery and how to reach out to new RSEs.

> Florian Goth has the pleasure of being grateful to all collaborators in this project for contributing their time and knowledge into this project over the past two years!

Reviewer 5: ```suggestion
Florian Goth has the pleasure of being grateful to all collaborators in this project for contributing their time and knowledge into this project over the span of  two years!
```

Reviewer 5: While "the past two" is correct now, it will read a little odd in 5 years time.

Reviewer 4: maybe just leave out _over the past two years_ since we are not done yet.

# Contribution details {-}
Heidi Seibold came up with the original idea for the deRSE23 workshop in Paderborn.
Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele, and Samantha Wittke organised the deRSE23 workshop.
We thank all the participants of this community workshop!
Toby Hodges conceptualised and organised the un-deRSE23 workshop together with Jan Philipp Thiele and Florian Goth.
We also thank all the participants of this follow-up community workshop!
Jeremy Cohen, Gerasimos Chourdakis, Magnus Hagdorn, Jean-Noël Grad, Jan Philipp Thiele, and Matthias Braun organised the deRSE24 workshop in Würzburg.
We are also grateful to the participants of this third community workshop!
Heidi Seibold, Jeremy Cohen, Florian Goth, Renato Alves, Jan Philipp Thiele,
Jan Linxweiler, Jean-Noël Grad, and Samantha Wittke contributed the initial draft.
Florian Goth supervised the project and did the project administration.
Jean-Noël Grad designed and implemented the software tooling.
We value the comments by Bernhard Rumpe and Wilhelm Hasselbring on previous drafts
of this work.
Everybody contributed to the final review and editing.

> Maybe it is obvious, but as you can guess from having various workshops

Reviewer 5: ```suggestion
```

> spread out over a two year period, and a total list of 23 authors

Reviewer 5: ```suggestion
```

> the CRediT system is far too generic to adequately describe the contributions of everybody.

Reviewer 4: I had to look up CRediT, I think we might need a reference here. Otherwise we should reformulate this sentence without mentioning it.

Author: Now we have a reference.

Reviewer 4: fabby

> the CRediT system [@Brand2015] is far too generic to adequately describe the contributions of everybody.

Reviewer 5: ```suggestion
The CRediT system [@Brand2015] is far too generic to adequately describe the contributions of everybody in various workshops, spread over a two year period.
```

> And while everybody contributed to the entirety of the script, some parts bear a personal handwriting.

Reviewer 4: ```suggestion
And while everybody contributed to the entirety of the script, some parts merit special mention.
```

Reviewer 2: I have the impression this was unintentionally marked as "resolved", without discussion.

Reviewer 2: Added again a suggestion in the latest state of the code: https://github.com/the-teachingRSE-project/competencies/pull/350/files#r1713336643

Resolving this thread.

> And while everybody contributed to the entirety of the script, some parts merit special mention.

Reviewer 5: While everybody contributed to the entirety of the script, some parts merit special mention.
```

> And while everybody contributed to the entirety of the script, some parts merit special mention.

Reviewer 5: ```suggestion
While everybody contributed to the entirety of the script, some parts merit special mention.
```

Renato Alves quickly jumped in to host the first deRSE23 workshop to take over from a sick organiser.
Matthias Braun contributed early versions of the specialisations and also contributed to the survey.
Leyla Jael Castro contributed to the initial draft of the example career path, and helped us in discussions on metadata.
Gerasimos Chourdakis' contributions to the paper are numerous but he especially helped us in writing first drafts for clarifying
the relationship of the RSE competencies to the SE competencies. He also designed the "Learning and teaching RSE" website [@learnandteach].
Simon Christ helped us with typesetting, contributed the competencies' symbols, and the spell-checker script.
Jeremy Cohen drafted the initial introduction and contributed the tables for RSEs in centralised RSE departments (Jean-Noël Grad made them render properly).
Stephan Druskat contributed parts on proper software citation and publication and sharpened various RSE specialisations.

> Fredo Erxleben contributed to early discussions of the paper and contributed the efforts done by Helmholtz.

Reviewer 4: ```suggestion
Fredo Erxleben contributed to early discussions of the paper and added the contributions by the Helmholtz Association.
```

Jean-Noël Grad contributed initial drafts for the ELIXIR framework and the section on the work of the HPC certification forum.
And, most likely, without him, we would not have a bibliography at all.
Magnus Hagdorn drafted and supervised the ethics and values section for an RSE and made sure that these values are reflected in the competencies of RSEs.
Toby Hodges contributed parts on the Carpentries, and helped steer the curriculum discussion.
Guido Juckeland contributed experiences from his first RSE course for students.
Dominic Kempf drafted the first version of the example curriculum.
Anna-Lena Lamprecht helped us to use proper wording and made us aware, where we were misusing established SE terminology.

> Jan Linxweiler drafted various RSE specialisations and made sure that clean coding techniques have their importance.

Reviewer 4: ```suggestion
Jan Linxweiler drafted various RSE specialisations and made sure that clean coding techniques got their due recognition.
```

Frank Löffler rewrote numerous parts to be actually legible, and helped us in preparing for the last mile of a de-RSE position paper.
Michele Martone wrote the first draft of the environmental sustainability section.
Moritz Schwarzmeier drafted the categorisation of the specialisations.
Heidi Seibold contributed the idea and started everything.
Jan Philipp Thiele drafted initial parts of the technical pillar of the RSE competencies, and represented the project on numerous discussions.
Harald von Waldow contributed to initial drafts of the Masters program and contributed his knowledge to the explainability of computer simulations.

> Samantha Wittke contributed the parts on CodeRefinery and how to reach out to beginning RSEs.

Reviewer 6: I think you probably want to say "new" rather than "beginning":
```suggestion
Samantha Wittke contributed the parts on CodeRefinery and how to reach out to new RSEs.
```
...or, if you wanted to extend this further, you maybe could say "_...and how to reach out to new RSEs who have recently joined the community._"?
